src/config/settings.default.js

```diff
@@ -1,6 +1,14 @@
 export default {
+  // App settings.
   defaultLocale: "en-US",
+
+  // Backend settings.
+  MIN_PASSWORD_LENGTH: 8,
+
+  // Network settings.
   API_BASE_URL: "https://kudo-o-matic-staging.dokku.kabisa.io",
```

Comment:
Shouldn't this be in ENV variables?

Reply:
Nope, it's not a secret and can easily be overriden in other settings files when the production url is known.